frontend/public/style/_overrides.scss

```diff
@@ -46,6 +46,18 @@ form.pf-v5-c-form {
   }
 }
 
+.co-catalog-page__grid {
+  // Get rid of weird stretching of catalog grid items
+  .pf-v5-c-card__header-main {
+    flex: unset;
+
+    // When images are too wide, scale them instead of squishing
+    img {
+      object-fit: contain;
+    }
```

Comment:
Hi @logonoff. The logo `img` already has `object-fit: contain;` applied to it via `co-catalog--logo`  https://github.com/openshift/console/blob/master/frontend/public/style/_overrides.scss#L20

Also, I'm unable to reproduce the stretched logos in your screenshot (Data Grid, Fluentd, Flomesh) on a 4.17 cluster. What browser and platform are you seeing that behavior on?

Reply:
Hi @sg00dwin! I am using Firefox 126.0 on Fedora 40 (Wayland). The stretching does not appear to exist on Chrome 125.0.6422.76, also on Fedora 40. I am using a v4.15.9 cluster, but this issue persists on the latest commit in `master`.

Reply:
Ok my initial comment was incorrect.  This issue is in the Developer Catalog and it has a slightly different markup structure than the OperatorHub Catalog, so those `img`s don't receive `object-fit: contain`. 

Can you add a unique modifier class and apply your overrides to it, so it's scoped to the Dev Catalog?

Reply:
The issue also applies to the Samples catalog in addition to the Developer catalog, as well as the Helm chart catalog. 

These catalogs appear to share the [CatalogController](https://github.com/openshift/console/blob/master/frontend/packages/console-shared/src/components/catalog/CatalogController.tsx) component in common, so I have added a `co-catalog-controller` class to it.

Please let me know if this is sufficient to scope just the Dev Catalogs!

Reply:
It looks like there is already a css class `odc-catalog-tile`  that applies the Catalog tiles only in the Developer perspective. So I would recommend applying the rules to it and not introduce a new class.

```
.odc-catalog-tile {
  // Get rid of weird stretching of catalog grid items
  .pf-v5-c-card__header-main {
    flex: unset;
    // When images are too wide, scale them instead of squishing
    img {
      object-fit: contain;
    }
  }
}
```